packages/sit-onyx/src/components/OnyxButton/OnyxButton.stories.ts

```diff
@@ -1,8 +1,23 @@
 import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import checkSmall from "@sit-onyx/icons/check-small.svg?raw";
 import type { Meta, StoryObj } from "@storybook/vue3";
 import OnyxButton from "./OnyxButton.vue";
 import { BUTTON_VARIATIONS, BUTTON_TYPES, BUTTON_MODES } from "./types";
 
+const ALL_ICONS = import.meta.glob("../../../node_modules/@sit-onyx/icons/src/assets/*.svg", {
```

Comment:
We also have this for the OnyxIcon, right? Can we extract this to a utility so we can re-use for future icon props as well?

Reply:
You can use #337 once its merged :)